frontends/property/src/styles/main.scss

```
@@ -1,3 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@600&family=Roboto:wght@100&display=swap');
+$M:768px;
+
+@mixin MQ($canvas){
+  @if $canvas==S{
+      @media only screen and (max-width:$M){
+          @content;
+      }
+  }
+  @else if $canvas==M{
+      @media only screen and (min-width:$M+1){
+          @content;
+      }
+  }
+}
+
 main {
     padding: 1rem;
-}
\ No newline at end of file
+}
+
+.testimonials{
```

- better to style against section as section is reused across the app.

```suggestion
main {
   section {
        // common styles
        
        &--testimonials { // section--testimonials
            // put your styles here
        }
   }
}
```